### PR TITLE
Update stdlib timestamp version

### PR DESCRIPTION
**Pull request by MohamedSabthar**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -10,57 +10,57 @@ slf4jVersion=1.7.30
...
-stdlibSqlVersion=1.7.0-20230104-085400-d1215dd
+stdlibGraphqlVersion=1.6.0-20230206-193700-bf2bf81
+stdlibSqlVersion=1.7.0-20230206-193300-bd2b276
 
 # Stdlib Level 08
-stdlibPersistVersion=0.1.0-20230131-111200-bfa75c1
+stdlibPersistVersion=0.1.0-20230206-204100-ccae3bd
 
 # Persist Tool
 persistToolVersion=0.1.0-20230131-110500-a6848bb
```
Comment (niveathika) [inline gradle.properties:66]:
```suggestion
persistToolVersion=0.1.0-20230206-214100-3e19d34
```

File: gradle.properties [hunk 2]
```diff
@@ -10,57 +10,57 @@ slf4jVersion=1.7.30
...
+stdlibOAuth2Version=2.6.0-20230206-172600-68a4a9b
 
 # Stdlib Level 05
-stdlibHttpVersion=2.6.0-20230103-222200-7ece8c5
+stdlibHttpVersion=2.6.0-20230206-181000-39543f5
 
 # Stdlib Level 06
-stdlibGrpcVersion=1.6.0-20230118-122700-c4ad07f
+stdlibGrpcVersion=1.6.0-20230206-185900-934405d
 stdlibTransactionVersion=1.4.0-20230104-083800-b83d606
```
Comment (niveathika) [inline gradle.properties:53]: Lets update trx version too

Comment (MohamedSabthar) [reply gradle.properties]: Updated with [e196b6c](https://github.com/ballerina-platform/ballerina-distribution/pull/4046/commits/e196b6ca8ab24d310bd330b9332d429207ca0b93)

**Review by anupama-pathirage — APPROVED** [review]

**Review by niveathika — DISMISSED** [review]

File: gradle.properties [hunk 3]
```diff
@@ -77,7 +77,7 @@ graphqlVersion=0.3.0
 protocToolVersion=0.1.0-20230125-180900-96d7ca0
 
 # OpenAPI Module
-openapiVersion=1.4.0-20230201-133700-58d39ab
+openapiVersion=1.4.0-20221220-122000-d6440e1
```
Comment (chiranSachintha) [inline gradle.properties:80]: Is this correct? Are we releasing openapi with the standard libraries?

Comment (MohamedSabthar) [reply gradle.properties]: Fixed with [4e5ce63](https://github.com/ballerina-platform/ballerina-distribution/pull/4046/commits/4e5ce63297194248a125f95f853578952435b529)

**Review by chiranSachintha — APPROVED** [review]

**Pull request merged**